README.md

```diff
@@ -3,42 +3,50 @@
 This repo serves to backup my neovim config file. I ended up making this public
 again for convenience.
 
-## Installation.
-Get Vim Plug (https://github.com/junegunn/vim-plug)
+## Installation
+
+First install Neovim. For Ubuntu, run the following:
+```
+sudo add-apt-repository ppa:neovim-ppa/stable
+sudo apt-get install -y neovim
```

Comment:
Could you add a link to the Neovim installation wiki and say for other distros, refer to here

Reply:
Done